hack/olm-registry/olm-artifacts-template.yaml

```diff
@@ -500,6 +500,10 @@ objects:
               index: openshift_managed_audit_stage
               whiteList: \.log$
               sourceType: _json
+            - path: /host/var/log/audit/audit.log
+              index: rh_osd_selinux
```

Comment:
I think we should wait for https://issues.redhat.com/browse/OSD-17654 to be complete. The index name would most probably be different to what's here.